scene/scene-properties-expressions/src/main/resources/ScenePropertiesExpressions.fxml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<?import com.esri.arcgisruntime.mapping.view.SceneView?>
+
+<StackPane fx:controller="com.esri.samples.scene.ScenePropertiesExpressionsController" xmlns:fx="http://javafx.com/fxml"
+           stylesheets="/style.css">
+    <!--SDK SceneView-->
+    <SceneView fx:id="sceneView"/>
+    <!--Slider box-->
+    <VBox StackPane.alignment="TOP_LEFT" maxWidth="200" maxHeight="50" spacing="5" styleClass="pane">
+        <Label text="heading"/>
+        <Slider fx:id="headingSlider" min="0" max="360" majorTickUnit="90" showTickLabels="true"/>
```

Comment:
True, but I think the intuition that a circle is 360 might be more important. The user can try putting a negative number or look at the doc to figure that out.